### ISSUE TITLE
AvoidUsingObjectMapperAsALocalVariable should consider @PostConstruct annotation

### ISSUE THREAD

**Issue by dgroup** [dgroup/arch4u-pmd#81]
The rule should not give violation when the field was initiated in the method marked by `@PostConstruct` annotation
   ```java
   public class X {
     private ObjectMapper mapper;

     @PostConstruct
     public void init() {
         mapper = new ObjectMapper();
     }
     ...
   }
   ```
